docs/pages/divider.js

```diff
@@ -1,64 +1,388 @@
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+            {
+              name: 'Shanice',
+              src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+            },
+          ]}
+        />
+      )}
+    </Flex>
+  );
+
+  return (
+    <Flex direction="column" gap={10} width={300}>
+      <Block title="Home decor" url="https://i.ibb.co/121JJzC/stock7.jpg" locked/>
+      <Flex width="100%" justifyContent="center">
+        <Box width="50%">
+          <Divider />
+        </Box>
+      </Flex>
+      <Block title="Plants" url="https://i.ibb.co/FY2MKr5/stock6.jpg" />
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use Dividers to separate collections of content or create the appearance of containers."
+            defaultCode={`
+function Example() {
+  return (
+    <Flex width="100%" height="100%">
+      <Box paddingX={4}>
+        <Flex direction="column" gap={2} width={120}>
+        {["Public profile", "Account settings", "Home feed tuner", "Claim", "Permissions", "Notifications"].map((item, idx) => <Text underline={idx === 0}weight="bold" key={idx} size="md" wrap="none">{item}</Text>)}
+        </Flex>
+      </Box>
+      <Divider />
+      <Box marginStart={12}>
+        <Flex direction="column" gap={4}>
+          <Heading accessibilityLevel="none" size="md">Public profile</Heading>
+          <Avatar
+            size="lg"
+            src="https://i.ibb.co/ZfCZrY8/keerthi.jpg"
+            name="Keerthi"
+          />
+          <TextField
+            id="first"
+            onChange={() => {}}
+            label="First name"
+            value="Kate"
+          />
+          <TextField
+            id="last"
+            onChange={() => {}}
+            label="Last name"
+            value="Dommeti"
+          />
+        </Flex>
+      </Box>
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use Dividers for decoration or to separate grouped content."
+            defaultCode={`
+function Example() {
+  return (
+    <Flex width="80%">
+      <Box paddingX={4}>
+        <Heading accessibilityLevel="none" size="sm">Our mission</Heading>
+      </Box>
+      <Divider />
+      <Box paddingX={4}>
+        <Text size="md">Pinterest's mission is to bring everyone the inspiration to create a life they love.</Text>
+      </Box>
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Accessibility">
+        <MainSection.Subsection description="The Divider is not focusable and is treated as decorative. Screen readers on tab navigation don't announce Dividers but do announce them on left/right quick navigation." />
```

Comment:
nit: no article needed, just "Divider is not focusable…"

Reply:
DONE